aten/src/ATen/cpu/Utils.cpp

```diff
@@ -45,7 +45,7 @@ bool init_amx() {
     return false;
   }
 
-#if defined(__linux__) && !defined(__ANDROID__)
+#if defined(__linux__) && !defined(__ANDROID__) && !defined(__s390x__)
```

Comment:
I'm a bit disappointed there isn't a more specific feature flag we can use to check this. Seems like it's only defined on X86/X64 for the moment.